packages/auto_animated/example/pubspec.lock

```diff
@@ -164,5 +164,5 @@ packages:
     source: hosted
     version: "0.3.3"
 sdks:
-  dart: ">=2.17.0-0 <3.0.0"
+  dart: ">=2.17.1 <3.0.0"
```

Comment:
unrelated change?

Reply:
sorry, it happened because I did `flutter pub update`.